sdk/communication/azure-communication-identity/tests/utils.py

```diff
@@ -18,4 +19,24 @@ def is_token_expiration_within_allowed_deviation(
     expected_seconds = expected_token_expiration.total_seconds();
     time_difference = abs(expected_seconds - token_expiration_in_seconds)
     allowed_time_difference = expected_seconds * allowed_deviation
-    return time_difference < allowed_time_difference
\ No newline at end of file
+    return time_difference < allowed_time_difference
+
+def generate_teams_user_aad_token(self):
+    if self.is_playback():
+        teams_user_aad_token = "sanitized"
+        teams_user_oid = "sanitized"
+    else:
+        msal_app = PublicClientApplication(
+            client_id=self.m365_client_id,
+            authority="{}/{}".format(self.m365_aad_authority, self.m365_aad_tenant))
+        scopes = [
+            "https://auth.msft.communication.azure.com/Teams.ManageCalls",
+            "https://auth.msft.communication.azure.com/Teams.ManageChats"
+        ]
+        result = msal_app.acquire_token_by_username_password(username=self.msal_username, password=self.msal_password, scopes=scopes)
+        teams_user_aad_token = result["access_token"]
+        teams_user_oid = result["id_token_claims"]["oid"]
+    return teams_user_aad_token, teams_user_oid
+
+def skip_get_token_for_teams_user_test(self):
```

Comment:
maybe it's better to pass `self.skip_get_token_for_teams_user_tests` instead of `self`

Reply:
Thank you! I updated.